tests/conftest.py

```diff
@@ -3014,6 +3014,57 @@ def tomte_mock_analysis_finish(
     )
 
 
+@pytest.fixture(scope="function")
```

Comment:
Consider adding Tomte (or even NF) related fixtures in a directory in `tests/fixture_plugins` to make this conftest file less crowded and fixtures more organised.

Reply:
100% agree. After tomte is integrated I want to refactor all workflow fixtures since they are very repetitive and moving them to plugins. But I'll do this in a coming PR

Reply:
resolving